### PR TITLE
2016 07 mn delete thumbnails

### PR DESCRIPTION
**Pull request by MagdaN**
with this also easy thumbnail images are deleted after the deletion of a model instance

### CODE REVIEW

File: euth/projects/signals.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+from easy_thumbnails.files import get_thumbnailer
+
+from .models import Project
+
+
+@receiver(post_delete, sender=Project)
+def delete_images_for_Project(sender, instance, **kwargs):
```
Comment (slomo) [inline euth/projects/signals.py]: Doesn't this naming break the function naming convention?

File: euth/organisations/signals.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+from easy_thumbnails.files import get_thumbnailer
+
+from .models import Organisation
+
+
+@receiver(post_delete, sender=Organisation)
+def delete_images_for_Organusation(sender, instance, **kwargs):
```
Comment (slomo) [inline euth/organisations/signals.py]: Same as below and typo in function name

**Comment by MagdaN** [comment]
@keimlink do you think it would be better to use something like https://github.com/akolpakov/django-unused-media (the problem might be that this would also delete the thumbnails)

**Comment by keimlink** [comment]
Your solution looks good. 👍

**Pull request merged**